pace-util/pace/util/grid/generation.py

```diff
@@ -84,6 +85,7 @@ def __init__(
                 self.LON_OR_LAT_DIM: 2,
                 self.TILE_DIM: 6,
                 self.CARTESIAN_DIM: 3,
+                self.UNIT_X_DIM: 1,
```

Comment:
Could you use pace.util.X_DIM instead? I don't think there's a need to differentiate UNIT_X_DIM from X_DIM - we similarly don't differentiate X_DIM_COMPUTE from X_DIM_FULL. It's enough that it indicates which axis the dimension is along (which is X_DIM).

Reply:
Changed to `pace.util.X_DIM`